tuplex/core/src/TraceVisitor.cc

```diff
@@ -561,6 +575,17 @@ namespace tuplex {
         if(it == _symbols.end()) {
             _symbols.push_back(TraceItem(ti.value, id->_name));
         } else {
+            // check if there is type change in loop
+            for (auto & el : _symbolsTypeChangeStack) {
```

Comment:
const auto& el?

Reply:
el is being modified in this loop, so cannot use const here?